src/lib/model.js

```diff
@@ -55,21 +55,54 @@ const processGroups = data => data.reduce(
   , []);
 const groupUsage = (dateFrom, dateTo) => query.usageByGroup(dateFrom, dateTo).then(processGroups);
 
+const hours = (name, colour) => ({
+  name,
+  data: Array.from({ length: 24 }, (_, n) => ({ count: 0, hour: n })),
+  colour
+});
+const dayCounts = (from, to) => (
+  (start, end,
+   diff = Math.abs(Math.round(moment.duration(start.startOf('day').diff(end.endOf('day'))).as('days'))),
+   mod = diff % 7,
+   weeks = Math.floor(diff / 7),
+   startDay = start.day() || 7,
+   extraWeekEndDays = mod && [6, 7].filter(n => startDay <= n && (startDay + mod) > n).length,
+   we = (weeks * 2) + extraWeekEndDays
+  ) => [diff - we, we]
+)(moment(from, DATE_FORMAT), moment(to, DATE_FORMAT));
```

Comment:
Seems like the IIFE is still here. Did you forget to push?

Reply:
I was thinking of something like this:
```js
const dayCounts = (from, to) => {
  const start = moment(from, DATE_FORMAT);
  const end = moment(to, DATE_FORMAT);

  const diff = Math.abs(Math.round(moment.duration(start.startOf('day').diff(end.endOf('day'))).as('days')));

  const mod = diff % 7;
  const weeks = Math.floor(diff / 7);
  const startDay = start.day() || 7;
  const extraWeekEndDays = mod && [6, 7].filter(n => startDay <= n && (startDay + mod) > n).length;
  const we = (weeks * 2) + extraWeekEndDays;

  return [diff - we, we];
};
```

Reply:
Yup that's exactly what I'd done, but it seems to have been lost somewhere in the rebasing... bloody git

Reply:
Fixed